### ISSUE TITLE
How should I interpret the results?

### ISSUE THREAD

**Issue by gdk43** [broadinstitute/infercnv#209]
When comparing the last part of chromosomes, the higher matrix data value is observed in observation than in reference, and more is amplified in infer graph.
Can you tell me why the pre_cnv_regions.dat and I6 HMM, Bayesian intensities are there more amplified in references?
The last chromosome has about 10 geniuses. Can you do that because it's too small?

![image](https://user-images.githubusercontent.com/59091518/72433945-d0e21900-37dd-11ea-9223-ba6e627af364.png)

In summary, the infercnv graph and HMM_intensities graph of the last chromosome had conflicting results.

thank you

**Comment by GeorgescuC** [comment]
Hi @gdk43 ,

I am not sure I understand entirely the case you are describing with the quality of the figures, but when only 10 genes are kept on a chromosome for the analysis, it is unlikely that anything can be said about cnvs in that chromosome, as there is too much noise to tell the signal out.

Regards,
Christophe.

**Comment by gdk43** [comment]
Hi Christophe,

Thanks for the reply. As you said, the number of genes seems to be the problem.

Best,

gkd43
